src/crawlee/crawlers/_basic/_basic_crawler.py

```diff
@@ -134,6 +134,11 @@ class _BasicCrawlerOptions(TypedDict):
     configure_logging: NotRequired[bool]
     """If True, the crawler will set up logging infrastructure automatically."""
 
+    use_table_logs: NotRequired[bool]
```

Comment:
I'm thinking, wouldn't it be better to have something like `statistics_log_format: Literal["table", "inline"]`? I think that most people won't know what a "table log" is...

Reply:
I like this idea